tests/tests/test-class-admin.php

```diff
@@ -380,26 +377,18 @@ public function test_filter_role_caps() {
 	 * @requires PHPUnit 5.7
 	 */
 	public function test_ajax_filters() {
-		$user = new \WP_User( get_current_user_id() );
-
 		$this->_setRole( 'subscriber' );
 
+		$user = new \WP_User( get_current_user_id() );
```

Comment:
The `$this->_setRole()` creates a new user with the required role and authenticates it so we should call `get_current_user_id()` only after it has been done.